### ISSUE TITLE
Config.toml no create

### ISSUE THREAD

**Issue by AkashiHidaka** [ethereum/go-ethereum#19863]
Hi there,

please note that this is an issue tracker reserved for bug reports and feature requests.

For general questions please use the gitter channel or the Ethereum stack exchange at https://ethereum.stackexchange.com.

#### System information

Geth version: `1.9.0 stable`
OS & Version: Ubuntu 16.04
Commit hash : (if `develop`)

#### Expected behaviour
When run config.toml make it generate empty file and show error.
INFO [07-19|00:20:44.030] Maximum peer count                       ETH=50 LES=0 total=50
INFO [07-19|00:20:44.030] Smartcard socket not found, disabling    err="stat /run/pcscd/pcscd.comm: no such file or directory"
toml: cannot marshal nil *params.TrustedCheckpoint

#### Actual behaviour
INFO [07-19|00:20:44.030] Maximum peer count                       ETH=50 LES=0 total=50
INFO [07-19|00:20:44.030] Smartcard socket not found, disabling    err="stat /run/pcscd/pcscd.comm: no such file or directory"
toml: cannot marshal nil *params.TrustedCheckpoint

#### Steps to reproduce the behaviour
I used new version and I think in there occur same issue.

#### Backtrace
I don't know about it.
````
[backtrace]
````

**Comment by rjl493456442** [comment]
Please try to build Geth from the latest master branch. This issue is already fixed by #19825

**Comment by chenyu1990** [comment]
```
Geth
Version: 1.9.9-stable
Git Commit: 017449971e1e9e220efcd97d3313a0e27f47003b
Git Commit Date: 20191206
Architecture: amd64
Protocol Versions: [64 63]
Go Version: go1.13.5
Operating System: linux
GOPATH=/data/go
GOROOT=/home/travis/.gimme/versions/go1.13.5.linux.amd64
root@dbcoin:/data/ethereum# geth --datadir /data/ethereum --cache 4096 --rpc --rp
cport 8545 --rpcaddr 127.0.0.1 --rpcapi "eth,personal" --allow-insecure-unlock du
mpconfig
INFO [12-28|18:58:05.603] Maximum peer count                       ETH=50 LES=0 t
otal=50
INFO [12-28|18:58:05.603] Smartcard socket not found, disabling    err="stat /run
/pcscd/pcscd.comm: no such file or directory"
ERROR[12-28|18:58:05.603] Failed to enumerate USB devices          hub=ledger ven
dor=11415 failcount=1 err="failed to initialize libusb: libusb: unknown error [co
de -99]"
ERROR[12-28|18:58:05.603] Failed to enumerate USB devices          hub=trezor ven
dor=21324 failcount=1 err="failed to initialize libusb: libusb: unknown error [co
de -99]"
ERROR[12-28|18:58:05.604] Failed to enumerate USB devices          hub=trezor ven
dor=4617  failcount=1 err="failed to initialize libusb: libusb: unknown error [co
de -99]"
ERROR[12-28|18:58:05.604] Failed to enumerate USB devices          hub=ledger ven
dor=11415 failcount=2 err="failed to initialize libusb: libusb: unknown error [co
de -99]"
ERROR[12-28|18:58:05.604] Failed to enumerate USB devices          hub=trezor ven
dor=21324 failcount=2 err="failed to initialize libusb: libusb: unknown error [co
de -99]"

```
